apps/build.sh

```diff
@@ -58,6 +57,16 @@ if [ -f $pbc ] ; then
 fi
 
 docker_login
+buildx_create_args="--driver-opt image=moby/buildkit:${BUILDKIT_VERSION} --use"
+if [ -d /usr/local/share/ca-certificates ] ; then
+	# We need to pass these certs into the buildkit container
+	bkconf="/buildkit.toml"
+	cat >${bkconf} <<EOF
+[registry."${hub_fio}"]
+ca=["/etc/ssl/certs/ca-certificates.crt"]
```

Comment:
I guess it cannot be specified via a CLI param?

Reply:
Not according to what I read

Reply:
`ca=["/etc/ssl/certs/ca-certificates.crt"]` but we check `if [ -d /usr/local/share/ca-certificates ]`, I mean we check different directory than use?

Reply:
the `load_extra_certs` takes the certs and produces an updated `/etc/ssl/certs/ca-certificates.crt` - so this more about making the container use the same certs as this "host"

Reply:
ok, a bit subtle.